testkit/src/main/java/org/apache/calcite/sql/test/SqlOperatorFixture.java

```diff
@@ -96,7 +96,7 @@ public interface SqlOperatorFixture extends AutoCloseable {
   // Error messages when an invalid time unit is given as
   // input to extract for a particular input type.
   String INVALID_EXTRACT_UNIT_CONVERTLET_ERROR =
-      "Extract.*from.*type data is not supported";
+      "Was not expecting value.*for enumeration.*in this context";
```

Comment:
This is to fix `SqlOperatorTest.testExtractIntervalDayTime`, the full error message is `Was not expecting value 'DOY' for enumeration 'org.apache.calcite.avatica.util.TimeUnit' in this context`.

I think the previous error message was wrong before, because I find that the exception check logic is introduced by CALCITE-1987 in 2017, and has remained unchanged, and the previous error message is introduced by CALCITE-4885 in 2021.